src/ProgressMeter.jl

```diff
@@ -580,11 +580,21 @@ function printvalues!(p::AbstractProgress, showvalues; color = :normal, truncate
     p.numprintedvalues = 0
 
     for (name, value) in showvalues
-        msg = "\n  " * lpad(string(name) * ": ", maxwidth+2+1) * string(value)
+        string_value = "\e[0m" * if value isa Union{String, SubString{String}}
+            value
+        else
+            repr("text/plain", value; context=IOContext(PipeBuffer(), :color => true))
```

Comment:
Is this repr the cause of the explicit quotes showing up?

Reply:
Yes. Trying to `show` it like in the previous commit causes it to show up as `AnnotatedString("    julia", ...)`